usl_models/usl_models/flood_ml/metastore.py

```diff
@@ -126,4 +191,4 @@ def _get_simulation_doc(
             urllib.parse.unquote(sim_name),
             safe=(),
         )
-    )
+    )
```

Comment:
You should configure your editor to place newlines at the end of files, as decreed by the laws of UNIX.

Reply:
Will do.